### PR TITLE
[375283] Available balance at the fiscal year level is calculated correctly (thunderjet) (TaaS)

### PR DESCRIPTION
**Pull request by dnovikau**
[C375283](https://foliotest.testrail.io/index.php?/cases/view/375283)
***
[Allure report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5621/allure)
***

Screenshot 2023-11-23 at 16 12 06

### CODE REVIEW

File: cypress/e2e/finance/fiscalYears/calulate-available-balance-for-negative-amount.cy.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        cy.login(testData.user.username, testData.user.password, {
+          path: TopMenu.fiscalYearPath,
+          waiter: FiscalYears.waitLoading,
+        });
+      });
+    });
+
+    after('Delete test data', () => {
+      cy.getAdminToken().then(() => {
+        Users.deleteViaApi(testData.user.userId);
```
Comment (ernestyedigaryan1) [inline cypress/e2e/finance/fiscalYears/calulate-available-balance-for-negative-amount.cy.js:119]: Please also delete other test datas, like: `Organizations`, `Orders`, `Invoices` ...

File: cypress/support/fragments/finance/fiscalYears/fiscalYearDetails.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
+import {
+  Button,
+  HTML,
+  KeyValue,
+  Link,
```
Comment (ernestyedigaryan1) [inline cypress/support/fragments/finance/fiscalYears/fiscalYearDetails.js:5]: Please remove unused import

File: cypress/support/fragments/finance/fiscalYears/fiscalYearDetails.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  MultiColumnListCell,
+  MultiColumnListRow,
+  PaneHeader,
+  Section,
+  including,
+} from '../../../../../interactors';
+
+const fiscalYearDetailsSection = Section({ id: 'pane-fiscal-year-details' });
+const fiscalYearDetailsHeader = PaneHeader({ id: 'paneHeaderpane-fiscal-year-details' });
+const actionsButton = fiscalYearDetailsHeader.find(Button('Actions'));
```
Comment (ernestyedigaryan1) [inline cypress/support/fragments/finance/fiscalYears/fiscalYearDetails.js:15]: Do we need this `actionsButton` here? It's not used in this file

**Review by ernestyedigaryan1 — CHANGES_REQUESTED** [review]

File: cypress/e2e/finance/fiscalYears/calulate-available-balance-for-negative-amount.cy.js (hunk 1 above)
Comment (dnovikau) [reply cypress/e2e/finance/fiscalYears/calulate-available-balance-for-negative-amount.cy.js:119]: When Invoice is turned to Paid status system creates transaction. If there are any transaction created it is not possible to delete related objects.

File: cypress/support/fragments/finance/fiscalYears/fiscalYearDetails.js (hunk 2 above)
Comment (dnovikau) [reply cypress/support/fragments/finance/fiscalYears/fiscalYearDetails.js:5]: Done

File: cypress/support/fragments/finance/fiscalYears/fiscalYearDetails.js (hunk 3 above)
Comment (dnovikau) [reply cypress/support/fragments/finance/fiscalYears/fiscalYearDetails.js:15]: Done

**Review by ernestyedigaryan1 — APPROVED** [review]

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**